header.php

```diff
@@ -56,7 +56,9 @@
   <section id="main-container">
 
   <!-- start content -->
-  <header id="header">
+  <header id="header" class="container">
+    <div class="row">
+      <div class="col col2">
```

Comment:
this is fixed in master from a PR of mine last week, but will clash with this markup as it is different